src/protocol/ipa/mod.rs

```diff
@@ -1265,7 +1238,7 @@ pub mod tests {
                     records_sent: 36_090,
                     bytes_sent: 133_776,
                     indexed_prss: 74_046,
-                    seq_prss: 1098,
+                    seq_prss: 1132,
```

Comment:
This one is weird. I changed `ShuffleRevealStep::*Permutation` to `ShuffleRevealPermutationStep::*` due to clippy errors. That affected as_ref() strings to change and ultimately caused this regression. I verified that reverting these as_ref() strings back fixed the issue. We should be okay since bytes_sent is unchanged, but I'm not sure whether seq_prss (or indexed_prss) is a useful metric to track?